src/osp/Active/SysAreaAssociate.cpp

```diff
@@ -199,8 +199,7 @@ void SysAreaAssociate::floating_origin_translate(Vector3 translation)
 }
 
 
-int SysAreaAssociate::sat_activate(universe::Satellite sat,
-                                   universe::UCompActivatable &satAct)
+int SysAreaAssociate::sat_activate(universe::Satellite sat)
 {
 
     if (m_activatedSats.find(sat) != m_activatedSats.end())
```

Comment:
Got a lot of left over code in this function? Maybe remove commented out code and instead put more inline documentaitons.

Reply:
8 lines below this, you have "satType".

What is "satType" supposed to represent? a type_id ?

Reply:
This function will be gone on my next PR. `satType` is a UCompType struct. It determines the type of a Satellite by storing an TypeSatIndex.